.travis.yml

```diff
@@ -1,60 +1,74 @@
-sudo: required
-dist: trusty
+dist: xenial
 compiler:
   - gcc
   - clang
 language: c
-before_install:
-  - sudo apt-get update -qq
-  - sudo apt-get install -qq --no-install-recommends
-      iptables-dev
-      libatasmart-dev
-      libcap-dev
-      libcurl4-gnutls-dev
-      libdbi0-dev
-      libesmtp-dev
-      libganglia1-dev
-      libgcrypt11-dev
-      libglib2.0-dev
-      libgps-dev
-      libhiredis-dev
-      libi2c-dev
-      libldap2-dev
-      libltdl-dev
-      liblua5.2-dev
-      liblvm2-dev
-      libmemcached-dev
-      libmicrohttpd-dev
-      libmnl-dev
-      libmodbus-dev
-      libmosquitto0-dev
-      libmysqlclient-dev
-      libnotify-dev
-      libopenipmi-dev
-      liboping-dev
-      libow-dev
-      libpcap-dev
-      libperl-dev
-      libpq-dev
-      libprotobuf-c0-dev
-      librabbitmq-dev
-      librdkafka-dev
-      librrd-dev
-      libsensors4-dev
-      libsigrok-dev
-      libsnmp-dev
-      libstatgrab-dev
-      libtokyocabinet-dev
-      libtokyotyrant-dev
-      libudev-dev
-      libupsclient-dev
-      libvarnish-dev
-      libvirt-dev
-      libxml2-dev
-      libyajl-dev
-      linux-libc-dev
-      perl
-      protobuf-c-compiler
-      python-dev
-# VERBOSE=1 dumps the contents of the test suites logs to standard output.
-script: sh build.sh && ./configure --with-libmongoc=own && make distcheck VERBOSE=1
+
+before_script: autoreconf -vif
+
+script:
+  - sh build.sh
+  - ./configure --enable-debug
+  # VERBOSE=1 dumps the contents of the test suites logs to standard output.
+  # -fsanitize=address enables https://github.com/google/sanitizers/wiki/AddressSanitizer
+  # -ldl works around a dlsym error when tsan is enabled in this environment.
+  - make check VERBOSE=1 CFLAGS=-fsanitize=address LDFLAGS="-ldl -fsanitize=address"
+  - make distcheck
+
+addons:
+  apt:
+    packages:
+    - autotools-dev
+    - iptables-dev
+    - libatasmart-dev
+    - libcap-dev
+    - libcurl4-gnutls-dev
+    - libdbi0-dev
+    - libesmtp-dev
+    - libganglia1-dev
+    - libgcrypt11-dev
+    - libglib2.0-dev
+    - libgps-dev
+    - libhiredis-dev
+    - libi2c-dev
+    - libldap2-dev
+    - libltdl-dev
+    - liblua50-dev
+    - liblua5.1-0-dev
+    - liblua5.2-dev
+    - libmemcached-dev
+    - libmicrohttpd-dev
+    - libmnl-dev
+    - libmodbus-dev
```

Comment:
`libmongoc-dev`?

Reply:
My goal was to make this identical to upstream, but sure, I can add it.